packages/scatter-plot-panel/src/panel.vue

```diff
@@ -0,0 +1,353 @@
+						limit: '-1',
+						filter: props.filter,
+						fields: [props.xAxis, props.yAxis, props.series],
+						sort: [props.series, props.xAxis],
+					}
+				});
+
+				if (props.series) {
+					let category_data: Record<string, any> = {};
+					response.data.data.forEach((item: Record<string, any>) => {
+						const category = formatTitle(get(item, props.series));
+						const x_value = get(item, props.xAxis);
+						const y_value = get(item, props.yAxis);
+						if(!(category in category_data)){
+							category_data[category] = [];
+						}
+						category_data[category].push([type == 'datetime' ? new Date(x_value):x_value, y_value]);
+					});
+					categories = Object.keys(category_data);
+					series = categories.map((category: string) => {
+						return {
+							name: category,
+							data: category_data[category],
+						};
+					});
+				} else {
+					series = [
+						{
+							name: formatTitle(props.collection),
+							data: response.data.data.map((item: Record<string, any>) => {
+								return [get(item, props.xAxis), get(item, props.yAxis)];
+							}),
+						},
+					];
+				}
+
+				const isSparkline = props.width < 12 || props.height < 10;
+
+				const axisStyle = {
+					fontFamily: 'var(--theme--fonts--sans--font-family)',
+					foreColor: 'var(--theme--foreground-subdued)',
+					fontWeight: 600,
+					fontSize: '10px',
+				};
+
+				let xLabels: Record<string, any> = {
+					show: ['both', 'xAxis'].includes(props.showAxisLabels),
+					offsetY: -4,
+					style: axisStyle,
+				};
+
+				if(type == 'datetime'){
+					xLabels.datetimeFormatter = {
+						year: 'yyyy',
+						month: 'MMM \'yy',
+						day: 'dd MMM',
+						hour: 'HH:mm'
+					};
+				} else {
+					xLabels.formatter = function(val: string) {
+						return xField.type == 'integer' ? parseInt(val) : parseFloat(val).toFixed(1);
+					};
+				}
+
+				chart.value = new ApexCharts(chartEl.value, {
+					colors: props.series || props.color == null ? [] : [props.color],
+					chart: {
+						type: 'scatter',
+						animation: {
+							enabled: false,
+						},
+						height: '100%',
+						width: '100%',
+						dropShadow: {
+							enabled: false,
+						},
+						toolbar: {
+							show: false,
+						},
+						selection: {
+							enabled: false,
+							fill: {
+								opacity: 0.5
+							},
+							stroke: {
+								opacity: 0
+							},
+						},
+						zoom: {
+							enabled: false,
+						},
+						fontFamily: 'var(--theme--fonts--sans--font-family)',
+						foreColor: 'var(--theme--foreground-subdued)',
+						sparkline: {
+							enabled: isSparkline,
+						},
+					},
+					series: series,
+					markers: {
+						size: 8,
+						fillOpacity: 0.5,
+						strokeOpacity: 0,
+						hover: {
+							size: 8,
+						},
+					},
+					tooltip: {
+						enabled: showToolTip,
+						x: {
+							formatter: (value: any) => {
+								let formattedValue = type == 'datetime' ? format(value, (xField.type == 'date' ? "MMM d" : "PPp")) : value > 10000
```

Comment:
Make sure to be consistent with the date format: use `xField.type == 'date' ? "MMM d" : "MMM d, p"` or `xField.type == 'date' ? "PP" : "PPp"`